src/libraries/System.Private.CoreLib/src/System/IndexOfAnyValues/IndexOfAny2CharValues.cs

```diff
@@ -14,7 +14,7 @@ internal sealed class IndexOfAny2CharValue<TShouldUsePacked> : IndexOfAnyValues<
         public IndexOfAny2CharValue(char value0, char value1) =>
             (_e0, _e1) = (value0, value1);
 
-        internal override char[] GetValues() => new[] { _e0 };
+        internal override char[] GetValues() => new[] { _e0, _e1 };
```

Comment:
I assume this was a typo in the first commit?

Reply:
Correct. This method is only used by the debug view, and we don't have tests for it specifically.
I'll add a test for this in a separate PR.